.github/xml-feed/generate-xml-feed.js

```diff
@@ -0,0 +1,310 @@
+const { XMLParser, XMLBuilder, XMLValidator } = require('fast-xml-parser');
+const fs = require('fs');
+const jsdom = require('jsdom');
+const { JSDOM } = jsdom;
+
+const Sheets = {
+  COMMUNITIES: 'communities',
+  SALES_OFFICES: 'sales-offices',
+  INVENTORY: 'inventory',
+  MODELS: 'models',
+  HOME_PLANS: 'home-plans',
+  CORPORATION: 'corporation',
+  SUBDIVISION: 'subdivision',
+};
+
+let salesOfficehelper = [];
+async function fetchWorkbook(sheetNames) {
+  const sheets = Array.isArray(sheetNames) ? sheetNames : [sheetNames];
+  const url = `https://main--hubblehomes-com--aemsites.hlx.live/data/hubblehomes.json?${sheets.map((sheetName) => `sheet=${sheetName}`).join('&')}`;
+  const request = await fetch(url);
+  if (request.ok) {
+    return request.json();
+  }
+  throw new Error('Failed to fetch workbook');
+}
+
+async function addInventoryData(model, community) {
+  const inventoryData = await fetchWorkbook(Sheets.INVENTORY);
+  let inventoryXml = '';
+  for (const inventoryhome of inventoryData.data) {
+    if (inventoryhome.community.toLowerCase() === community.toLowerCase() && inventoryhome['model name'].toLowerCase() === model.toLowerCase()) {
+      inventoryXml += ` <Spec Type="SingleFamily">
```

Comment:
If any text data inserted into the XML (e.g., description, Caption, etc.) contains characters like &, <, or >, could it break the XML structure.? Do the special characters need to be escaped?